beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -785,6 +785,28 @@ impl<T: BeaconChainTypes> Worker<T> {
 
                 verified_block
             }
+            Err(e @ BlockError::SlashableProposal) => {
+                warn!(
+                    self.log,
+                    "Received equivocating block from peer";
+                    "error" => ?e
+                );
+                /* punish peer for submitting an equivocation, but not too harshly as honest peers may conceivably forward equivocating blocks to us from time to time */
+                self.gossip_penalize_peer(
+                    peer_id,
+                    PeerAction::LowToleranceError,
```

Comment:
If the idea's to not downscore peers too harshly this should probably be a `MidToleranceError` or `HighToleranceError`, I'd probably lean `HighToleranceError`

Reply:
`MidToleranceError` seems to make more sense from a DoS perspective, but good catch.

Reply:
I think MidTolerance is good. Honest peers should only forward at most 1 slashable block, so should be able to take the hit from a mid (or even a low) tolerance downscore